documentation/modules/configuring/ref-storage-persistent.adoc

```diff
@@ -5,36 +5,48 @@
 [id='ref-persistent-storage-{context}']
 = Persistent storage
 
-Persistent storage uses {K8sPersistentVolumeClaims} to provision persistent volumes for storing data.
-Persistent Volume Claims can be used to provision volumes of many different types, depending on the {K8SStorageClass} which will provision the volume.
-The data types which can be used with persistent volume claims include many types of SAN storage as well as {K8sLocalPersistentVolumes}.
+[role="_abstract"]
+Persistent data storage retains data even when a pod that uses the storage restarts.
+Persistent data storage uses a storage framework called persistent volumes (PV). 
+PVs are storage resources, independent of the pods that use them.
+Pod configuration uses {K8sPersistentVolumeClaims} (PVCs) to make storage requests on the PVs.
+The PVC requests the amount of storage required when a pod is being created.
+The underlying storage infrastructure of the PV does not need to be understood. 
+If a PV matches the storage criteria, the PVC is bound to the PV.
 
-To use persistent storage, the `type` has to be set to `persistent-claim`.
-Persistent storage supports additional configuration options:
+Because of its permanent nature, persistent storage is recommended for production. 
```

Comment:
To avoid confusion here ... 
* `type: persistent-claim` is recommended for production for ZooKeeper
* `type: jbod` with one or more `type: persistent-claim` volumes is recommended for production for Kafka